From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @bandersmsft, has been notified to review your proposed change.

bandersmsft commented:
@4c74356b41 Thanks for catching my copy and paste error.
@PRmerger @jim-parker This update is good to merge.
#sign-off

troyhopwood committed:
Updated Pre-Configured Devices

Removed mention of frequency from pre-configured device models and added mention that behaviors are controlled from JavaScript.

danieldobalian committed:
refactored the v2 toc

VaijanathB committed:
Resolving the confusion regarding storage keys exposed. The guidance is just to avoid checking in this file

troyhopwood committed:
Changing max devices from 1,000 to 20,000

Commit:
fixing url in image

danieldobalian committed:
Adding platforms to .NET

mimig1 committed:
Cosmos DB: Meta cleanup manager:jhubbard->kfile

parakhj committed:
Adding Common AAD article

parakhj committed:
Minor changes

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into freshness

Commit:
freshness

markgalioto committed:
Adjusting the Backup TOC for Troubleshooting information

Commit:
Fixing github version of table

Commit:
fixing table

rjagiewich committed:
Merge pull request #37131 from danieldobalian/master

refactored the v2 toc

Commit:
updating ms.topic per metadata sweeps

mimig1 committed:
Cosmos DB: Removing connected-service article for now

rjagiewich committed:
Merge pull request #37490 from cynthn/table-dv2

Table dv2

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#6580

Updates parameter switches with correct dashes, not em dashes

zhiweiwangmsft committed:
Remove showresult

PRMerger18 committed:
Merge pull request #37197 from VaijanathB/terraform2

Resolving the confusion regarding storage keys exposed. …

femsulu committed:
(Forums Team) Fix step five #6877

Step five should read Start-AzureRMAutomationRunbook $RBParams

iainfoulds committed:
[VM] Add code fencing

v-shils committed:
Merge pull request #37451 from mimig1/04092018metacheck

Cosmos DB: Meta cleanup manager:jhubbard->kfile

Commit:
Merge pull request #37496 from femsulu/patch-8

(Forums Team) Fix step five #6877

v-shils committed:
Merge pull request #37488 from markgalioto/apr102018

Adjusting the Backup TOC for Troubleshooting information

martincoetzer committed:
Update azure-security.md

Remove AD-FS reference for tokens

vasivara-MSFT committed:
(ForumsTeam) Adding ']' to upgraded workspace payload 

Json is invalid missing ']' - Please take a look at this https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-alerts-actions#sample-payload'

martincoetzer committed:
Update azure-security-identity-management-best-practices.md

Adding reference to Password Hash Sync and Pass-through authentication. Changing link for AD-FS deployment

PRMerger12 committed:
Merge pull request #37495 from zhiweiwangmsft/patch-30

Remove showresult

PRMerger14 committed:
Merge pull request #37499 from martincoetzer/patch-6

Update azure-security-identity-management-best-practices.md

femsulu committed:
(Forums Team) Revert change in PR#37496

Step five should read Start-AzureRMAutomationRunbook @RBParams (confirmed by George Wallace)

PRMerger14 committed:
Merge pull request #37498 from martincoetzer/patch-5

Update azure-security.md

Commit:
Zones redo

PRMerger17 committed:
Merge pull request #37502 from femsulu/patch-9

(Forums Team) Revert change in PR#37496

v-shils committed:
Merge pull request #37202 from troyhopwood/master

Updating per March Release

jodebrui committed:
small clarification

rothja committed:
Fixing SQL VM Linux scoped  TOC and overview to match Windows mirror

mimig1 committed:
Cosmos DB: Added new graph partitioning doc to exp TOC

v-shils committed:
Merge pull request #37491 from Blackmist/freshness

Freshness

GitHubber17 committed:
Merge pull request #37494 from iainfoulds/patch-6

[VM] Tag a VM - updated dash format

Commit:
freshness

PRMerger10 committed:
Merge pull request #37506 from jodebrui/patch-13

small clarification

PRMerger13 committed:
Merge pull request #37510 from Blackmist/freshness

freshness

rothja committed:
Fixing two windows references to Linux

PRMerger14 committed:
Merge pull request #37500 from vasivara-MSFT/patch-2

(ForumsTeam) invalid Json MicrosoftDocs/azure-docs#6829

rmca14 committed:
Merge pull request #37457 from parakhj/commonaad

Adding Common AAD article

mimig1 committed:
added links to SQL and graph partitioning docs

ramankumarlive committed:
removed the content for blob and files

GitHubber17 committed:
Merge pull request #37508 from rothja/linuxtoc2

Fixing SQL VM Linux scoped  TOC and overview to match Windows mirror

Commit:
Merge pull request #37514 from ramankumarlive/patch-5

removed the content for blob and files

GitHubber17 committed:
Merge pull request #37492 from mimig1/04102018toc

Cosmos DB: Removing connected-service article for now

GitHubber17 committed:
Merge pull request #37509 from mimig1/04102018datamigrationtool

Cosmos DB: Added new graph partitioning doc to exp TOC

douglaslMS committed:
Added 3 sets of FAQ updates from lengthy email thread with CSA.

douglaslMS committed:
Made one small revision.

Commit:
bug

douglaslMS committed:
Made a few Acrolinx fixes.

PRMerger19 committed:
Merge pull request #37518 from douglaslMS/sync-updates-rdavis

Added 3 sets of FAQ updates from lengthy email thread with CSA.

Ja-Dunn committed:
Merge pull request #37507 from cynthn/zones-redo

Zones redo

iainfoulds committed:
[VMSS] Fixes MicrosoftDocs/azure-docs#6797

Updating zone-redundant CLI version for cross-platform support and consistency with script example.

Commit:
Merge pull request #37520 from iainfoulds/patch-7

[VMSS] Availability Zones CLI command example

vladvino committed:
Edits and added soap bindings

iainfoulds committed:
[VM] VM agent update command - additional examples

Updating sed command in other examples, per https://github.com/MicrosoftDocs/azure-docs/pull/4848

Commit:
metadata

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger15 committed:
Merge pull request #37523 from iainfoulds/patch-1

[VM] VM agent update command

Ja-Dunn committed:
Merge pull request #37521 from vladvino/patch-10

Edits and added soap bindings

v-alje committed:
Merge pull request #37527 from MicrosoftDocs/FromPublicRepo

From public repo

douglaslMS committed:
Added PM clarifications to my recent updates.

Commit:
Merge pull request #37524 from cjgronlund/dsvm-metadata

metadata updates

Commit:
Merge pull request #37530 from douglaslMS/sync-updates-rdavis2

Added PM clarifications to my recent updates.
